src/server/ua_server_internal.h

```diff
@@ -191,6 +191,9 @@ struct UA_Server {
     /* Statistics */
     UA_SecureChannelStatistics secureChannelStatistics;
     UA_ServerDiagnosticsSummaryDataType serverDiagnosticsSummary;
+
+    /* Transaction for certificate management */
+    UA_Transaction *transaction;
```

Comment:
Is there only one kind of transaction specified for OPC UA or should this one be renamed (and also the structure)?